frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/capacity-card/capacity-card.tsx

```diff
@@ -42,22 +41,36 @@ const QueriesMatchingCapacityView: QueryMapType = {
     CAPACITY_USAGE_QUERIES[StorageDashboardQuery.STORAGE_CEPH_CAPACITY_REQUESTED_QUERY],
     CAPACITY_USAGE_QUERIES[StorageDashboardQuery.STORAGE_CEPH_CAPACITY_USED_QUERY],
   ],
-  [CapacityViewType.CAPACITY_VMS_VS_PODS]: [
-    CAPACITY_USAGE_QUERIES[StorageDashboardQuery.STORAGE_CEPH_CAPACITY_PODS_QUERY],
-    CAPACITY_USAGE_QUERIES[StorageDashboardQuery.STORAGE_CEPH_CAPACITY_VMS_QUERY],
-  ],
 };
 
-export const CapacityCard: React.FC<DashboardItemProps> = ({
+const getItems = (plugin: DashboardStorageExtension[], flags: FlagsObject) =>
```

Comment:
Same as in #2476, nit: please use more sensible parameter names.

Will be fixed in follow-up.